From 0760b1fd5f16a19475a25084be6e935a9ce7cbb5 Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fabian@ritter-vogt.de>
Date: Wed, 13 Mar 2024 16:35:47 +0100
Subject: [PATCH] gtkconfig.upd: Set Version=6

Make sure it's actually run by kded6.

BUG: 482763
(cherry picked from commit 5b8327530ee62a257f9f12d8b419b445beec345b)
---
 kconf_update/gtkconfig.upd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kconf_update/gtkconfig.upd b/kconf_update/gtkconfig.upd
index 7583199..570f296 100644
--- a/kconf_update/gtkconfig.upd
+++ b/kconf_update/gtkconfig.upd
@@ -1,4 +1,4 @@
-Version=5
+Version=6
 
 Id=gtk_theme
 Script=gtk_theme
-- 
2.43.2

